src/main/io/osd.h

```diff
@@ -157,13 +160,13 @@ typedef struct osdConfig_s {
     uint8_t ahMaxPitch;
     uint8_t ahMaxRoll;
     bool enabled_stats[OSD_STAT_COUNT];
+    int8_t esc_temp_alarm;
+    int16_t esc_rpm_alarm;
```

Comment:
Not 100% sure, but there seems to be a 1-byte padding inserted after `esc_temp_alarm` to ensure 2-byte alignment for `esc_rpm_alarm`.

Reply:
According to my count the 32bit boundary is between `esc_temp_alarm` and `esc_temp_alarm`. Is there a way to verify this?